contributor-guide/modules/ROOT/pages/testing/boost-test-matrix.adoc

```diff
@@ -1,286 +1,178 @@
 = Boost Test Matrix
-:idprefix:
-:idseparator: -
+:navtitle: Boost Test Matrix
 
-*Legacy content, currently being updated.*
+It's straightforward to run regression tests on your Boost clone, by utilizing some scripts available for the purpose.
```

Comment:
That's the thing. The page about regression tests is "Testing". Not "Boost Test Matrix".